framework/source/class/qx/util/Wheel.js

```diff
@@ -39,6 +39,11 @@ qx.Bootstrap.define("qx.util.Wheel", {
      */
     FACTOR: 1,
 
+    /**
+     * Is the Wheel actually a touchpad ?
+     * @internal
```

Comment:
I don't know, but I thought this was done in order to omit the inclusion in docs and tell the reader that it it is not a reliable public API.
I really just applied the same thing than for the other variables in the class.